recipes/openmp/bld.bat

```diff
@@ -0,0 +1,20 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+    -DCMAKE_BUILD_TYPE="Release" ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX:PATH=%LIBRARY_PREFIX% ^
+    %SRC_DIR%
+
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
+nmake install
+if errorlevel 1 exit 1
+
+mkdir -p %LIBRARY_PREFIX%/lib/clang/%PKG_VERSION%/include
+# Standalone libomp build doesn't put omp.h in clang's default search path
```

Comment:
Batch comments don't start with `#` unfortunately. They instead use `::` or `rem`. Personally I prefer `::`, but either should be fine.